tests/positive/files/link.go

```diff
@@ -360,7 +360,6 @@ func ForceHardLinkCreation() types.Test {
 		ConfigMinVersion: configMinVersion,
 	}
 }
-
```

Comment:
Don't think this line removal was intentional.

Reply:
Fixed

Reply:
I only see a single blank line in between the two functions. @bgilbert - maybe refresh?

Reply:
It was removed in the first commit and re-added in the second.

Reply:
ahh, yes I see that if I look at the first commit by itself.

Reply:
Fixed